pywbem/_logging.py

```diff
@@ -14,371 +14,303 @@
-  drive WBEM operations (see :ref:`WBEM operations`). Log
-  entries are created at INFO log level. This logs the parameters of each
-  request and the CIMObjects/exceptions in each method repoonse.
-
-* ``pywbem.http`` logs HTTP requests and responses between the pywbem client
-  and WBEM servers, at the info level. This logs the http request data and
-  response data. This named logger is also defined to create log entries when
-  logging is at the INFO log level.
-
-To output log records for one of the defined named loggers, either
-:meth:`~pywbem._logging.PywbemLoggers.create_loggers` or the
-:meth:`~pywbem._logging.PywbemLoggers.create_logger` should be used to define
-the characteristics of the named logger.
-
-* :meth:`~pywbem._logging.PywbemLoggers.create_loggers` creates one or more
-  loggers from
-  a string input that defines the component name and characteristics of each
-  logger.  This allows other tools like CLIs that use pywbem to create the
-  pywbem known logs with minimal work from command line or config file input.
-
-* :meth:`~pywbem._logging.PywbemLoggers.create_logger` creates one or more
-  loggers from
-  parameter inputs that define the component name and characteristics of each
-  logger.
-
-These functions save the logger definitions in the
-:class:`~pywbem._logging.PywbemLoggers` class that
-is used by the logging functions in the recorder so trying to create the named
-loggers independently of this code may cause issues.
-
-The pywbem loggers are based on two parameters (log destination, log detail)
-that determine if the logs for the logger name are created,
-how much information is inserted, and the log destination.  This
-extends the python logging facility to include the log_detail parameter which
-defines whether all the information defined or only a limited size is output.
-This is used with pywbem because the logs on operation and http responses can be
-very large.
-
-The code that executes the loggers call the function get_logger(..) to get a
-logger from a defined logger name.  If that logger has not yet been defined
-in PywbemLoggers, an entry will be added with the default parameters.
-"""
-from __future__ import absolute_import
+Pywbem logging is used to record information passing between the pywbem client
+and WBEM servers but not as a general recorder for errors, state, etc. within
+pywbem. In effect it is a trace tool.
+Pywbem errors are generally passed to the pywbem API user as Python
+exceptions rather than being recorded in a log by a pywbem logger.
 
-import logging
-import six
+Pywbem logging supports two :class:`~py:logging.Logger` objects
+(named loggers):
 
-__all__ = ['PywbemLoggers',
-           'LOG_OPS_CALLS_NAME', 'LOG_HTTP_NAME', 'LOG_COMPONENTS',
-           'LOG_DESTINATIONS', 'LOG_DETAIL_LEVELS', 'DEFAULT_LOG_DETAIL_LEVEL',
-           'DEFAULT_MAX_LOG_ENTRY_SIZE', 'DEFAULT_LOG_DESTINATION']
+* ``pywbem.api`` - logs user-issued calls to :class:`~pywbem:WBEMConnection`
```

Comment:
Incorrect reference to WBEMConnection. Change ":" to "."

Reply:
Done